src/documents/tutorial-userInterfaceOptions/ContentPropertyHighContrast.md

```diff
@@ -0,0 +1,60 @@
+---
+title: CSS "Content" Property in Contrast Modes
+layout: default
+category: Tutorials
+---
+
+The CSS `content` property can be used to add additional content to your webpage using CSS. Often it is used to add a character, symbol, or font icon before or after some content on webpage. For example it can be used to add vertical bars to separate items on a horizontal navigation list, for example: ![A horizontal navigation list with blue text on grey background.](../images/tutorial-uio-content-default.png)
```

Comment:
The image might look better if it was on a separate line. Also something to indicate that it is an image and not text content would help too. Not sure if we have any conventions for this yet though.